pkg/controller/plan/adapter/vsphere/builder.go

```diff
@@ -723,6 +828,10 @@ func (r *Builder) mapDisks(vm *model.VM, vmRef ref.Ref, persistentVolumeClaims [
 				},
 			},
 		}
+		if disk.Shared {
+			kubevirtDisk.Shareable = ptr.To(true)
```

Comment:
This line is not needed. it's only needed one one disk is attached multiple times to the same vm.